backend/corpora/common/utils/db_session.py

```diff
@@ -52,10 +52,11 @@ def db_session_manager(**kwargs):
             session.commit()
         else:
             session.expire_all()
-    except SQLAlchemyError:
-        session.rollback()
-        msg = "Failed to commit."
-        logger.exception(msg)
-        raise CorporaException(msg)
+    except Exception as e:
```

Comment:
Fixed the exception handling as it was hiding non-`SQLAlchemyError`s. (e.g. when the secret couldn't be read)